package.json

```diff
@@ -16,9 +16,9 @@
     "lint:fix": "npm run lint -- --fix",
     "prettier": "npx prettier --config .prettierrc.json -w ./src",
     "prettier:check": "npx prettier --config .prettierrc.json --check ./src",
-    "prepare": "cd ../.. && husky install frontend/app/.husky",
+    "prepare": "cd ../.. && husky install .husky",
     "test": "yarn run test-jest",
-    "test-jest": "REACT_APP_IS_TEST=true NODE_ENV=test yarn jest --coverage --no-cache --transformIgnorePatterns \"./frontend/app/svg/\""
+    "test-jest": "REACT_APP_IS_TEST=true NODE_ENV=test yarn jest --coverage --maxWorkers 2 --no-cache --transformIgnorePatterns \"./svg/\""
```

Comment:
nice does this make the tests faster?

Reply:
Yeah, it fixes aa max thread error I usually get when the tests run for long.